<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>215</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Blinker</family>
    <pointsize>26</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Guess My Number</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Titlelabel">
       <property name="font">
        <font>
         <family>Blinker</family>
         <pointsize>26</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Guess My Number</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <widget class="QSpinBox" name="NumberspinBox">
        <property name="font">
         <font>
          <family>Blinker</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
       <widget class="QPushButton" name="GuessButton">
        <property name="font">
         <font>
          <family>Blinker</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Guess</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StartOverButton">
        <property name="font">
         <font>
          <family>Blinker</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Over</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Messagelabel">
       <property name="font">
        <font>
         <family>Blinker</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Titlelabel_2">
       <property name="font">
        <font>
         <family>Blinker</family>
         <pointsize>10</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Created by Aboeita</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
